src/app/collective-rewards/rewards/backers/Rewards.tsx

```diff
@@ -135,10 +136,14 @@ export const Rewards: FC<RewardsProps> = ({ builder, tokens }) => {
             isDetailedView ? 'order-4' : 'order-3',
           )}
         />
-        {/* 
-      // Removed until RBI% is introduced (TOK-610)
-        <BackerAllTimeShare gauges={gauges} tokens={tokens} />
-      */}
+        <RBI
+          backer={builder}
+          tokens={tokens}
+          className={cn(
+            'flex flex-col flex-none mb-[46px] w-[214px]',
```

Comment:
These classes look the same as for the other cards. Should we store it in a const?

Reply:
I think that @jurajpiar  did some changes in the latest commits that were introduced to the NFT feature. Once we merge those to develop we can check it :)

Reply:
I don't want this comment to be a blocker, so we can merge the PR as it is right now 👍🏼